Pinta.Core/Classes/AsyncEffectRenderer.cs

```diff
@@ -110,7 +103,28 @@ internal double Progress {
 		}
 	}
 
-	internal void Start (
+	/// <summary>
+	/// Retrieves the union of all tiles that have finished rendering since the
+	/// last time this method was called, and resets the updated area.
+	/// </summary>
+	/// <returns>
+	/// True if there was an updated area to retrieve, otherwise false.
+	/// </returns>
+	internal bool TryGetAndResetUpdatedBounds (out RectangleI bounds)
```

Comment:
Maybe something like "consume" or "steal" could be more concise than GetAndReset here?